source/features/emphasize-draft-pr-label.css

```diff
@@ -2,12 +2,17 @@
 
 /* stylelint-disable-next-line media-feature-name-no-vendor-prefix -- It's the only cross-browser media query */
 @media (-webkit-min-device-pixel-ratio: 2) {
-	[rgh-emphasize-draft-pr-label] .js-issue-row :is(
-	/* Repo PR lists */
-	[aria-label='Open draft pull request'],
-	/* Global PR lists */
-	[aria-label='Draft Pull Request']
-) svg {
+	[rgh-emphasize-draft-pr-label]
+		.js-issue-row
+		:is(
+			[aria-label='Open draft pull request'],
+			[aria-label='Draft Pull Request']
+		)
```

Comment:
Might be worth checking if these are still applied on those two pages.

Also make sure comments are not lost

Reply:
> Might be worth checking if these are still applied on those two pages.

Yeah, it still needs these.